updater/lib/dependabot/updater/operations/refresh_version_update_pull_request.rb

```diff
@@ -58,8 +58,12 @@ def initialize(service:, job:, dependency_snapshot:, error_handler:)
         def perform
           Dependabot.logger.info("Starting update job for #{job.source.repo}")
           Dependabot.logger.info("Checking and updating versions pull requests...")
+
           dependency = dependencies.last
 
+          # Raise an error if the package manager version is unsupported
+          dependency_snapshot.package_manager&.raise_if_unsupported!
```

Comment:
When we have `unsupported error` for the package manager, we don't want to go over each dependency, instead we want to raise unsupported error for the whole process and handle error properly.